presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduUtil.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kudu;
+
+import io.airlift.log.Logger;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.security.UserGroupInformation;
+import org.apache.kudu.client.KuduClient;
+
+import java.io.IOException;
+import java.security.PrivilegedExceptionAction;
+
+public class KuduUtil
+{
+    private static final Logger log = Logger.get(KuduUtil.class);
+
+    private KuduUtil()
+    {
+        // not allowed to be called to initialize instance
+    }
+
+    /**
+     * Initialize kerberos authentication
+     */
+    static void initKerberosENV(String principal, String keytab, boolean debugEnabled)
+    {
+        try {
+            Configuration conf = new Configuration(false);
+            conf.set("hadoop.security.authentication", "kerberos");
+            if (debugEnabled) {
+                System.setProperty("sun.security.krb5.debug", "true");
+            }
+            UserGroupInformation.setConfiguration(conf);
```

Comment:
Do we need to add hadoop dependencies ? How about reusing `KerberosAuthentication` in presto plugiin toolkit ?